packages/components/src/toolbar/toolbar/style.scss

```diff
@@ -13,9 +13,15 @@
 
 		& > .components-toolbar-group {
 			border-right: none;
+			background: transparent;
```

Comment:
This background color change could be a problem for existing usages of the `unstyled` variant. There are a couple of safer ways to achieve this from the consumer, but I think the simplest way for now would be to pass your own CSS class to the `ToolbarGroup` instance(s) you want transparent.

```
.my-toolbar-group {
  background: transparent;
}

<ToolbarGroup className="my-toolbar-group" />
```